SpatialGDK/Documentation/content/map-travel.md

```diff
@@ -4,7 +4,8 @@ Before reading this topic, make sure you are familiar with the Unreal documentat
 
 > Note: This topic is intended for advanced users only.
 
-## `APlayerController::ClientTravel`
+## ClientTravel
+`APlayerController::ClientTravel`
```

Comment:
Either remove `APlayerController::ClientTravel` or put it in paranthesis "ClientTravel (`APlayerController::ClientTravel`)"

Reply:
It won't work as an in-page link like that. I tried lots of different methods and thought this one will make it easier for now.
Am I OK to get this in so we can get a latest version in place on testing? Then fix later?